### DISCUSSION TITLE
How to hide WalletConnect QR code?

### DISCUSSION THREAD

**Discussion by mqklin** [WalletConnect/walletconnect-monorepo#2985] [Developer Support] [answered]
Is there an option to remove WalletConnect as an option from Web3Modal? As it doesn't work for our dApp in every mobile wallet 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/9456433/1619ad35-19e1-47cf-a1b2-25dddb02fd2a)

**Comment by ryespresso** [comment]
This would be nice. Keep it more minimalist with the option to bring up the wallet connect QR code.

**Comment by glitch-txs** [comment, marked as answer]
Will add a more minimalist style on Web3Modal v3 :) (coming soon!)
